src/gui/owncloudgui.cpp

```diff
@@ -1025,8 +1020,7 @@ void ownCloudGui::slotShutdown()
     // that saving the geometries happens ASAP during a OS shutdown
 
     // those do delete on close
-    if (!_settingsDialog.isNull())
-        _settingsDialog->close();
+    _settingsDialog->close();
```

Comment:
Is it safe to ->close()M if .isNull() ?

Reply:
The dialogue is initialised in the constructor and should always exist.